doc/whats_new.rst

```diff
@@ -53,6 +53,8 @@ BUG
 
     - Fix channel name pick in :func:`mne.Evoked.get_peak` method by `Alex Gramfort`_
 
+    - Fix drop percentages to take into account ``ignore`` option in :func:`mne.viz.plot_drop_log` and :func:`mne.Epochs.plot_drop_log` by `Eric Larson`_.
```

Comment:
slightly orthogonal question but why are both functions public? Shouldn't we have only the Epochs function in the public API?